drivers/staging/android/lowmemorykiller_tng.c

```diff
@@ -302,7 +302,7 @@ void print_obituary(struct task_struct *doomed,
 		     global_node_page_state(NR_SLAB_UNRECLAIMABLE) *
 		     (long)(PAGE_SIZE / 1024),
 		     gfp_mask,
-		     global_node_page_state(NR_INDIRECTLY_RECLAIMABLE_BYTES)
+		     global_node_page_state(NR_KERNEL_MISC_RECLAIMABLE)
 		     / 1024,
```

Comment:
`NR_KERNEL_MISC_RECLAIMABLE` is in pages, not bytes (as per b29940c1abd7a4c3abeb926df0a5ec84d6902d47)

Reply:
Converted from pages to memory as with other entries, adjusted.